bumping-android-version-code.md

```diff
@@ -0,0 +1,23 @@
+# Bumping Android Version Code
+
+#### 🤔 What Happened
+
+If you're submitting your Android app to Google Play Store using the `expo upload:android` command, you may encouter the following error:
```

Comment:
we may want to avoid repeating the error message in the doc so that we don't have to update in multiple places if we want to change it, and risk it going out of sync. we can summarize the error here instead.

Reply:
I totally agree :)